lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -272,7 +272,6 @@ def self.create_automation_object(name, attrs, options = {})
     options[:instance_name] = name
 
     options[:attrs] = create_ae_attrs(attrs, name, options[:vmdb_object])
-    options[:message] = options[:attrs][:message]
```

Comment:
@tinaafitz @d-m-u 
https://github.com/ManageIQ/manageiq/blob/master/app/models/service_template/filter.rb#L6
It passes message using the options[:attrs][:message]

https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/buttons.rb#L505

It uses a different field. We need to make them consistent so that message can be passed in the same way for both the calls.